ViewControllers/MostPopularViewController.swift

```diff
@@ -36,11 +41,11 @@ class MostPopularViewController: UIViewController, UICollectionViewDelegate {
         
         loadMovieData()
         
-//        I did it in an attemt to make loadMovieData method reusable in willDisplayCell method. I'm not sure it's right, though.
+//      I did it in an attemt to make loadMovieData method reusable in willDisplayCell method. I'm not sure it's right, though.
         DispatchQueue.main.async {
             self.contentView.collectionView.scrollToItem(at: IndexPath(item: self.itemInViewIndex, section: 0), at: UICollectionView.ScrollPosition.centeredHorizontally, animated: false)
             self.configureWithData(index: self.itemInViewIndex)
-            self.fadeIn()
+            self.fade(Fading.fadeIn)
```

Comment:
Usually with enums we can take advantage of swift's type inference and write like this:
```swift
self.fade(.fadeIn)
```
Not a change request here, just a FYI